src/main/resources/org/jenkinsci/plugins/workflow/multibranch/template/ConfigDrivenWorkflowMultiBranchProjectFactory/help-pipelinePath.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Relative default location of pipeline_template.
```

Comment:
Same suggestion as above 😄 
```suggestion
    Relative default location of <code>pipeline_template</code>. This is useful when you have some repositories in an organization where you do not want to customize any values for the template or you cannot add a Config File to the repository.
```